src/main/java/qtlog/util/ConfigManager.java

```diff
@@ -0,0 +1,62 @@
+package qtlog.util;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigManager {
+    private static final String CONFIG_FILE = "src/main/resources/config.properties";
+
+    public static String readLogPath() {
+        Properties properties = new Properties();
+        try (FileInputStream input = new FileInputStream(CONFIG_FILE)) {
+            properties.load(input);
+            return properties.getProperty("logPath", "");
```

Comment:
Lets assume the default arc log location unless configured. On windows I guess it should be something like this:
```java
FileSystemView.getFileSystemView().getDefaultDirectory().getPath() + "\\Guild Wars 2\\addons\\arcdps\\arcdps.cbtlogs\\"
```

Reply:
Left it in to test configuring the logpath manually